### PR TITLE
docs: ja contents translation and latest contents syncing

### PR DESCRIPTION
**Pull request by kazupon**

### CODE REVIEW

File: content/fr/docs/4.directory-structure/3.components.md [hunk 1]
```diff
@@ -122,39 +122,36 @@ Si on a des composants dans des répertoires imbriqués comme ceci:
...
 ```
 
 Nous recommandons d'utiliser le nom du répertoire dans le nom du fichier pour plus de clarté.
 
 ```html
-<BaseFooButton />
+<BaseFooCustomButton />
 ```
 
-Cependant, si l'on souhaite garder `Button.vue`, on peut utiliser l'option `prefix` dans le fichier de configuration `nuxt.config.js` pour ajouter un préfixe à tous les composants dans un répertoire donné.
```
Comment (kazupon) [inline content/fr/docs/4.directory-structure/3.components.md:134]: I had this part translated by a translation service. I'm not sure if the translation is correct or not, so I would like you to check it for me. 🙏

File: content/fr/docs/4.directory-structure/3.components.md [hunk 2]
```diff
@@ -122,39 +122,36 @@ Si on a des composants dans des répertoires imbriqués comme ceci:
...
-Ainsi, on pourra utiliser `BaseButton` dans le template or lieu de `Button` sans pour autant devoir changer le nom du fichier `Button.vue`.
+Et maintenant nous pouvons utiliser `<CustomButton />` au lieu de `<BaseFooCustomButton />`.
 
 ```html{}[pages/index.vue]
-<FooButton />
+<CustomButton />
 ```
 
+::alert{type="next"}
+Voir [la propriété components](/docs/configuration-glossary/configuration-components) pour d'autres méthodes de contrôle du nom des composants.
```
Comment (kazupon) [inline content/fr/docs/4.directory-structure/3.components.md:152]: Same here, I had this part translated by a translation service. I'm not sure if the translation is correct or not, so I would like you to check it for me. 🙏

File: content/fr/docs/4.directory-structure/9.pages.md [hunk 3]
```diff
@@ -244,6 +244,18 @@ export default {
...
+Comme la propriété `key` qui peut être utilisée sur les composants Vue dans les modèles comme une indication pour le DOM virtuel, cette propriété permet à la valeur de la clé d'être définie à partir de la page elle-même (plutôt que le composant parent).
+
+Par défaut dans Nuxt, cette valeur sera le `$route.path`, ce qui signifie que naviguer vers une route différente assurera la création d'un composant de page propre. Logiquement équivalent à :
+
+```html
+<router-view :key="$route.path" />
+```
+
+La propriété peut être une `String` ou une `Function` qui prend la route comme premier argument.
+
```
Comment (kazupon) [inline content/fr/docs/4.directory-structure/9.pages.md:258]: Same here, I had this part translated by a translation service. I'm not sure if the translation is correct or not, so I would like you to check it for me. 🙏

File: content/fr/docs/4.directory-structure/3.components.md (hunk 1 above)
Comment (smarroufin) [reply content/fr/docs/4.directory-structure/3.components.md:134]: I'd say

> Si nous voulons le déclarer comme `<CustomButton />` tout en gardant la structure de répertoire, nous pouvons ajouter le répertoire de `CustomButton.vue` dans `nuxt.config.js`.

File: content/fr/docs/4.directory-structure/3.components.md (hunk 2 above)
Comment (smarroufin) [reply content/fr/docs/4.directory-structure/3.components.md:152]: LGTM

File: content/fr/docs/4.directory-structure/9.pages.md (hunk 3 above)
Comment (smarroufin) [reply content/fr/docs/4.directory-structure/9.pages.md:258]:
```md
### key

Comme la propriété `key` qui peut être utilisée sur les composants Vue dans les modèles comme une indication pour le DOM virtuel, cette propriété permet à la valeur de la clé d'être définie à partir de la page elle-même (plutôt que le composant parent).

Par défaut dans Nuxt, cette valeur sera `$route.path`, ce qui signifie que naviguer vers une route différente assurera la création d'un composant de page propre. Logiquement équivalent à :

```html
<router-view :key="$route.path" />
```

La propriété peut être une `String` ou une `Function` qui prend la route comme premier argument.
```

**Comment by kazupon** [comment]
@smarroufin 
Thank you for your reviewing!
I've just updated.
https://github.com/nuxt/nuxtjs.org/pull/1810/commits/15e49c364f6ed152f099a3117bb3f96964b3d96f

**Pull request merged**